Packs/Venafi/Integrations/VenafiV2/VenafiV2.yml

```
@@ -1,309 +1,309 @@
+      description: An array of Organization Units or names.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.PublicKeyHash
+      description: The public key hash string. Available only when the certificate has a private key.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SKIKeyIdentifier
+      description: The generated Subject Key Identifier (SKI).
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SignatureAlgorithm
+      description: The signature algorithm for signing the certificate.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SignatureAlgorithmOID
+      description: The Signature Object ID for signing the certificate.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.StoreAdded
+      description: The Date Time stamp when the private key was added to the store.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SubjectAltNameDNS
+      description: An array of Domain Name System (DNS) SANs.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SubjectAltNameEmail
+      description: An array of Email SANs. Based on RFC 822.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SubjectAltNameOtherNameUPN
+      description: An array of User Principal Name (UPN) SANs.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SubjectAltNameIPAddress
+      description: An array of IP address SANs.
+      type: string
+    - contextPath: Venafi.Certificate.CertificateDetails.SubjectAltNameURI
+      description: An array of Uniform Resource Indicator (URI) SANs.
+      type: string
+    - contextPath: Venafi.Certificate.CreatedBy
+      description: The object that initiated enrollment or provisioning changes. The default is Web SDK.
+      type: string
+    - contextPath: Venafi.Certificate.Origin
+      description: Filter by origin.
+      type: string
+    - contextPath: Venafi.Certificate.ProcessingDetails
+      description: 'Absent when the certificate is not currently processing in the Trust Protection Platform lifecycle: InError, InProcess, Stage, Status, TicketDN.'
+      type: dictionary
+    - contextPath: Venafi.Certificate.RenewalDetails
+      description: A list of certificate renewal information.
+      type: dictionary
+    - contextPath: Venafi.Certificate.ValidationDetails
+      description: 'A list of host identity information and the overall certificate validation state result. If no validation occurred, only the lastvalidationstateupdate field appears. All other validationdetails fields are absent.'
+      type: dictionary
+    description: Uses a certificate GUID to extract more details from the certificate store.
   runonce: false
   script: '-'
   type: python
   subtype: python3
   dockerimage: demisto/python3:3.10.14.92207
-fromversion: 5.5.0
+fromversion: 5.0.0
```

```suggestion
fromversion: 6.9.0
```
validate this is the lower default version